crates/biome_js_analyze/src/lint/suspicious/no_double_equals.rs

```diff
@@ -156,7 +156,7 @@ impl Rule for NoDoubleEquals {
 /// Rule's options
 #[derive(Clone, Debug, Deserializable, Eq, PartialEq, serde::Deserialize, serde::Serialize)]
 #[cfg_attr(feature = "schemars", derive(JsonSchema))]
-#[serde(rename_all = "camelCase", deny_unknown_fields)]
+#[serde(rename_all = "camelCase", deny_unknown_fields, default)]
```

Comment:
`serde(default)` is already specified on every field.
You should use one of them, not both.